src/os/bluestore/NVMEDevice.cc

```diff
@@ -801,6 +816,7 @@ void io_complete(void *t, const struct spdk_nvme_cpl *completion)
   assert(queue != NULL);
   assert(ctx != NULL);
   ++queue->completed_op_seq;
+  --queue->current_queue_depth;
```

Comment:
1）should this also supported in sync write and read?
2）should it be defined as atomic_t?